### DISCUSSION TITLE
3.2.0-beta03

### DISCUSSION THREAD

**Discussion by Him188** [open-ani/animeko#465] [Announcements]
- 修复有些资源搜不到的问题
- 修复有时候无法发送弹幕的问题
- 全新搜索页面
- Android 增加日志分享功能 #176

**Full Changelog**: https://github.com/open-ani/ani/compare/v3.2.0-beta02...v3.2.0-beta03

----

### 下载

[//]: # (注意, `checkLatestVersion` 有字符串处理, 修改标题和分隔符前务必查询)

[github-win-x64]: https://github.com/open-ani/ani/releases/download/v3.2.0-beta03/ani-3.2.0-beta03-windows-x86_64.zip

[github-mac-x64]: https://github.com/open-ani/ani/releases/download/v3.2.0-beta03/ani-3.2.0-beta03-macos-x86_64.dmg

[github-mac-aarch64]: https://github.com/open-ani/ani/releases/download/v3.2.0-beta03/ani-3.2.0-beta03-macos-aarch64.dmg

[github-android]: https://github.com/open-ani/ani/releases/download/v3.2.0-beta03/ani-3.2.0-beta03.apk

[cf-win-x64]: https://d.myani.org/v3.2.0-beta03/ani-3.2.0-beta03-windows-x86_64.zip

[cf-mac-x64]: https://d.myani.org/v3.2.0-beta03/ani-3.2.0-beta03-macos-x86_64.dmg

[cf-mac-aarch64]: https://d.myani.org/v3.2.0-beta03/ani-3.2.0-beta03-macos-aarch64.dmg

[cf-android]: https://d.myani.org/v3.2.0-beta03/ani-3.2.0-beta03.apk

[ghproxy-win-x64]: https://mirror.ghproxy.com/?q=https%3A%2F%2Fgithub.com%2Fopen-ani%2Fani%2Freleases%2Fdownload%2Fv3.2.0-beta03%2Fani-3.2.0-beta03-windows-x86_64.zip

[ghproxy-mac-x64]: https://mirror.ghproxy.com/?q=https%3A%2F%2Fgithub.com%2Fopen-ani%2Fani%2Freleases%2Fdownload%2Fv3.2.0-beta03%2Fani-3.2.0-beta03-macos-x86_64.dmg

[ghproxy-mac-aarch64]: https://mirror.ghproxy.com/?q=https%3A%2F%2Fgithub.com%2Fopen-ani%2Fani%2Freleases%2Fdownload%2Fv3.2.0-beta03%2Fani-3.2.0-beta03-macos-aarch64.dmg

[ghproxy-android]: https://mirror.ghproxy.com/?q=https%3A%2F%2Fgithub.com%2Fopen-ani%2Fani%2Freleases%2Fdownload%2Fv3.2.0-beta03%2Fani-3.2.0-beta03.apk

[qb-enhanced]: https://github.com/c0re100/qBittorrent-Enhanced-Edition/releases/latest

PC 目前默认只使用在线数据源。
若本机装有 qBittorrent 或 [qBittorrent Enhanced][qb-enhanced] (建议), 则可额外支持 BT 数据源。
可在设置中配置 qBittorrent 连接方式, 随后可在数据源设置中启用 BT 数据源。

PC 首次播放在线数据源时, 可能需要加载 10-30 秒。

| 操作系统                    | 全球                           | 中国大陆                                             |
|-------------------------|------------------------------|--------------------------------------------------|
| Windows x86_64          | [GitHub][github-win-x64]     | [主线][cf-win-x64] / [备线][ghproxy-win-x64]         |
| macOS x86_64 (Intel 芯片) | [GitHub][github-mac-x64]     | [主线][cf-mac-x64] / [备线][ghproxy-mac-x64]         |
| macOS aarch64 (M 系列芯片)  | [GitHub][github-mac-aarch64] | [主线][cf-mac-aarch64] / [备线][ghproxy-mac-aarch64] |
| Android APK aarch64     | [GitHub][github-android]     | [主线][cf-android] / [备线][ghproxy-android]         |

扫描二维码下载 Android 版本：

[github-android-qr]: https://github.com/open-ani/ani/releases/download/v3.2.0-beta03/ani-3.2.0-beta03.apk.github.qrcode.png

[cf-android-qr]: https://d.myani.org/v3.2.0-beta03/ani-3.2.0-beta03.apk.cloudflare.qrcode.png

| 全球                           | 中国大陆                         |
|------------------------------|------------------------------|
| ![GitHub][github-android-qr] | ![Cloudflare][cf-android-qr] |

## What's Changed
* feat: support extraFiles in ikaros data source for episode video attachment subtitles. by @li-guohao in https://github.com/open-ani/ani/pull/452
* fix: 视频播放器UI上的字幕列表不及时更新 by @li-guohao in https://github.com/open-ani/ani/pull/453
* fix: 无法正确设置 qBittorrent 分类的问题 by @he0119 in https://github.com/open-ani/ani/pull/456
* feat: 视频内置音轨切换 by @li-guohao in https://github.com/open-ani/ani/pull/455
* rewrite searchpage pt.1 by @StageGuard in https://github.com/open-ani/ani/pull/459
* Him188/simplify sourceset structure by @Him188 in https://github.com/open-ani/ani/pull/460

<hr /><em>This discussion was created from the release <a href='https://github.com/open-ani/ani/releases/tag/v3.2.0-beta03'>3.2.0-beta03</a>.</em>

**Comment by Him188** [comment]
这日志库怎么导致安装包变大了7M啊，才发现

**Reply by Him188** [reply]
加日志之前有8万行代码+一大堆依赖+几个图标+libtorrent+视频播放器才19M，这日志相当于这些的40%

**Reply by StageGuard** [reply]
可能是 room runtime 用到了 sqlite driver

**Reply by Him188** [reply]
确实，错怪它了
